README.md

```diff
@@ -70,3 +70,21 @@ All lead associations will be based on the domain of the experience being viewed
 <script id="ceros-marketo-munchkin-plugin" src="//sdk.ceros.com/marketo-munchkin/main-v0.js" accountId="AAA-111-BBB"></script>
 ```
 2. Replace the value of the accountId attribute with your marketo munchkin Id
+
+## The State of the Layer Plugin
+
+Allows people using the Ceros Studio's SDK pallet to tag components to enable them to both control, and be controlled by, parameters in the Experience's URL. It can be used to remember user's actions between pages or to trigger actions like showing layers or jumping to anchor points when the Experience is opened by the user.
```

Comment:
its essentially providing a framework for using the # as a means to click hotspots across page requests.

basically I create a hotspot that does some thing. tag it with "now-kiss=true".

then i can go to <viewurl>#now-kiss=true and that hotspot will get kissed.

tom has provided a way to use set-state to append the payload of a component to the url to give some interaction that will i suppose allow for a deep link to a non page load state of an experience etc etc

Tom might have to explain where the need comes from